libr/bin/format/mz/mz.c

```diff
@@ -78,6 +75,10 @@ struct r_bin_mz_segment_t * r_bin_mz_get_segments(
 		btree_add (&tree, (void *)&stack_segment, cmp_segs);
 	}
 
+	if (num_relocs * sizeof(*segments) ==  0) {
```

Comment:
The multiplication is innecesary



> On 22 Aug 2015, at 17:47, Álvaro Felipe Melchor <notifications@github.com> wrote:
> 
> In libr/bin/format/mz/mz.c:
> 
> > @@ -78,6 +75,10 @@ struct r_bin_mz_segment_t * r_bin_mz_get_segments(
> >  		btree_add (&tree, (void *)&stack_segment, cmp_segs);
> >  	}
> >  
> > +	if (num_relocs * sizeof(*segments) ==  0) {
> why? I could write if (num_relocs == 0) since sizeof will never be 0.
> 
> —
> Reply to this email directly or view it on GitHub.
>